ras_party/controllers/account_controller.py

```diff
@@ -591,15 +594,26 @@ def post_case_event(case_id, party_id, category='Default category message', desc
     return response.json()
 
 
-def request_cases_for_respondent(respondent_id):
+def request_cases_for_respondent(respondent_id, state=None):
     logger.debug('Retrieving cases for respondent', respondent_id=respondent_id)
     url = f'{current_app.config["RAS_CASE_SERVICE"]}/cases/partyid/{respondent_id}'
+    if state:
+        url = url + '?state=' + state
     response = Requests.get(url)
     response.raise_for_status()
     logger.debug('Successfully retrieved cases for respondent', respondent_id=respondent_id)
     return response.json()
 
 
+def set_bi_cases_for_party_to_inactionable(party_id):
+    logger.debug("Set respondent BI cases to inactionable", party_id=party_id)
+
+    cases = request_cases_for_respondent(party_id, state='ACTIONABLE')
+    if cases:
```

Comment:
Case service returning a 204 is annoying. Instead of having an if statement here it might be better to change request_cases_for_respondent to
`return response.json() if response.json() else []`?